homeassistant/components/random/binary_sensor.py

```diff
@@ -59,10 +59,9 @@ class RandomBinarySensor(BinarySensorEntity):
 
     def __init__(self, config: Mapping[str, Any], entry_id: str | None = None) -> None:
         """Initialize the Random binary sensor."""
-        self._attr_name = config.get(CONF_NAME)
+        self._attr_name = config[CONF_NAME]
         self._attr_device_class = config.get(CONF_DEVICE_CLASS)
```

Comment:
I'm new to the code base, forgive me... Just so I understand, you can change .get() to [] for CONF_NAME because it's guaranteed to be set by the PLATFORM_SCHEMA bit above, but still have to use get() for CONF_DEVICE_CLASS since it can not be set?